src/app/sb-admin/components/pages/sb-user/sbuser.component.ts

```diff
@@ -48,33 +51,50 @@ export class SbUserComponent implements OnInit {
             { field: "phone", header: "Phone" },
             { field: "status", header: "Status" },
             { field: "channel", header: "Channel" },
-          ];
+        ];
 
         this.statuses = [
             { label: 'ACTIVE', value: 'active' },
             { label: 'INACTIVE', value: 'inactive' }
         ];
 
-        this.organizations = [
-            { name: 'NULP', code: 'NULP' },
-            { name: 'SHIKSHALOKAM', code: 'SHIKSHALOKAM' },
-            { name: 'TEKDI_TEST', code: 'TEKDI_TEST' }
-        ];
     }
 
-    openNew() {
-        this.user = {};
-        this.submitted = false;
-        this.userDialog = true;
+    getUserList() {
+        const body = {
+            "request": {
+                "filters": {
+                    "rootOrgId": []
+                },
+                "limit": 1000
+            }
+        }
+        this.userService.getUserList(body).subscribe((Response) => {
+            this.users = Response.result.response.content
```

Comment:
Response => response
Variable name should be in small case

Reply:
done